chronicle-sandbox/src/main/java/net/openhft/chronicle/engine/client/internal/ClientWiredExcerptAppenderStateless.java

```diff
@@ -15,15 +15,15 @@
 
 public class ClientWiredExcerptAppenderStateless extends AbstractStatelessClient implements ExcerptAppender {
 
-    private final Bytes source = Bytes.elasticByteBuffer();
+    private final Bytes<?> source = Bytes.elasticByteBuffer();
     private final Wire wire;
     private ChronicleQueue queue;
     private long cid;
     private long lastWrittenIndex = -1;
 
     public ClientWiredExcerptAppenderStateless(ClientWiredChronicleQueueStateless queue,
                                                ClientWiredStatelessTcpConnectionHub hub,
-                                               Function<Bytes, Wire> wireWrapper) {
+                                               Function<Bytes<?>, Wire> wireWrapper) {
```

Comment:
Not related to this PR but strictly we should declare `Function<? super Bytes<?>, ? extends Wire>`

Reply:
Possibly, however I am not sure it's clearer